src/rmkit/input/events.cpy

```diff
@@ -133,6 +151,30 @@ namespace input:
               self.lifted = true
 
           break
+        case ABS_MT_WIDTH_MAJOR:
+          slots[slot].width_major = data.value
+          break
+        case ABS_MT_WIDTH_MINOR:
+          slots[slot].width_minor = data.value
+          break
```

Comment:
Do we actually get these WIDTH ones? If so we might be able to figure out palm just from these, since these are "size of the approaching tool" (i.e. not touching the screen).

Reply:
we don't